data/gegno/gegno intro missions.txt

```diff
@@ -439,6 +442,311 @@ mission "Giaru Gegno: Revisit"
+			`	You return to your ship and take off from the planet, but the Scin do not follow you any further.`
+				decline
+				
+			label "no thoughts"
+				`	The Scin stares at you for an uncomfortable length of time, then sighs. They reach into their armor and pull out a small firearm, raising it at you.`
+			choice
+				`	(Reach for your firearm.)`
+				`	(Raise your arms.)`
+			action
+				set "scin: provocative"
+			`As soon as you attempt to move, the first thing you notice is a blade around your neck, and a bunch of sand flying past you. The Scin casually marks another note on their device, then gets up and and starts heading back to their ship. Slowly turning your head, your eyes are met with the cold, expressionless look of another Scin's face, who then motions you to return to your ship. They follow you as you do, continuing to hold the blade to your throat. Two more Scin suddenly appear from the sand along the way, while the others you first met remain at their position. The armor of the Scin escorting you is considerably different from the rest, characterized by darker shades, sharper edges, and several gadgets.`
+			`	Once you are at your ship, they move their blade away from you neck. Getting a better look at it, the blade is curved like a sickle, as opposed to a sword, meaning there was little chance you could have avoided being decapitated if they chose to do so. You board your ship in a hurry, preparing to take off. It's clear you've overstayed your welcome.`
+				flee
+			
+			label "reach"
+			action
+				set "scin: provocative"
+			`	It is rather alarming that the Scin be comfortable in approaching you casually, and considering how the encounter with the Vi went, anything could happen next. You decide to reach for your pistol, but as soon as you do, the first thing you notice is a blade around your neck, and a bunch of sand flying past you. Slowly turning your head, your eyes are met with the cold, expressionless look of another Scin's face, who then motions you to return to your ship. They follow you as you do, continuing to hold the blade to your throat. Two more Scin suddenly appear from the sand along the way, while the others you first met remain at their position. The armor of the Scin escorting you is considerably different from the rest, characterized by darker shades, sharper edges, and several gadgets.`
+			`	Once you are at your ship, they move their blade away from you neck. Getting a better look at it, the blade is curved like a sickle, as opposed to a sword, meaning there was little chance you could have avoided being decapitated if they chose to do so. Boarding your ship in a hurry, you prepare to take off, and you notice the two Scin warships also begin to power up. It's clear you've overstayed your welcome.`
```

Comment:
```suggestion
			`	Once you are at your ship, they move their blade away from you neck. Getting a better look at it, the blade is shaped like a sickle, as opposed to a sword, meaning there was little chance you could have avoided being decapitated if they chose to do so. Boarding your ship in a hurry, you prepare to take off, and you notice the two Scin warships also begin to power up. It's clear you've overstayed your welcome.`
```
If you accept the above suggestion for describing the blade as curved when you feel it around your neck, then you could change this to not overuse the word `curved`.